index.ftd

```diff
@@ -918,6 +918,10 @@ integer logo-width: $common.logo-width
 integer logo-height: $common.logo-height
 optional string site-url: $common.site-url
 optional string site-name:
+optional body bio:
+boolean social: false
+optional ftd.color background-color: $inherited.colors.background.step-1
```

Comment:
```suggestion
```
@MeenuKumari28 don't add any CSS as a Property to any component, its not recommended in our best practices.